### PR TITLE
init-frontend

### PR DESCRIPTION
**Pull request by alisdg**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -443,3 +443,110 @@ modules.xml
...
+.node_repl_history
+
+# Output of 'npm pack'
+*.tgz
+
+# Yarn Integrity file
+.yarn-integrity
+
+# dotenv environment variables file
+.env
```
Comment (karimiehsan90) [inline .gitignore:506]: Don't add .env

File: .gitignore [hunk 2]
```diff
@@ -443,3 +443,110 @@ modules.xml
...
+dist
+
+# vuepress build output
+.vuepress/dist
+
+# Serverless directories
+.serverless
+
+# IDE / Editor
+.idea
```
Comment (karimiehsan90) [inline .gitignore:527]: It is duplicated. I added before.

File: ms_pro/package.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "ms_pro",
```
Comment (karimiehsan90) [inline ms_pro/package.json:2]: Please change name of directory and fontend project to "frontend" or "user-interface". Suppose we name NLP module like this module. How can I understand which module is frontend and which module is NLP?
Git convention for commits is in ordering. This commit is better to be "Initialize frontend module". Please do same its later

File: .gitignore (hunk 1 above)
Comment (alisdg) [reply .gitignore:506]: ok, mr :\

**Pull request merged**